### PR TITLE
Update "queue plan" button text

### PR DESCRIPTION
**Pull request by sdehm**
### What
Changed the action for starting a new plan to "Queue plan" to match the UI that we see.

### Why
Current UI does not seem to match the documentation.

### Screenshots

----------

### Merge Checklist
_If items do not apply to your changes, add (N/A) and mark them as complete._

#### Pull Request
- [x] One or more labels describe the type of change (e.g. clarification) and associated product (e.g. tfc).
- [x] (N/A) Description links to related pull requests or issues, if any.

#### Content
- [x] (N/A) Redirects have been added for moved, renamed, or deleted pages.
- [x] (N/A) Sidebar navigation files have been updated for added, deleted, reordered, or renamed pages.
- [x] (N/A) Links to related content where appropriate (e.g., API endpoints, permissions, etc.).
- [x] (N/A) Pages with related content are updated and link to this content when appropriate.
- [x] (N/A) New pages have metadata (page name and description) at the top.
- [x] (N/A) New images are 2048 px wide. They have HashiCorp standard annotation color (#F92672) and format (rectangle with rounded corners), blurred sensitive details (e.g. credentials, usernames, user icons), and descriptive alt text in the markdown for accessibility.
- [x] (N/A) New code blocks have the correct syntax and line breaks to eliminate horizontal scroll bars.
- [x] (N/A) UI elements (button names, page names, etc.) are bolded.
- [x] (N/A) The Vercel website preview successfully deployed.

#### Reviews
- [x] I or someone else reviewed the content for technical accuracy.
- [x] I or someone else reviewed the content for typos, punctuation, spelling, and grammar.

### CODE REVIEW

**Comment by laurapacilio** [comment]
Hi @sdehm - Thank you for submitting this :-) Two things:
- Are you using Terraform Cloud or Terraform Enterprise?
- Could you provide a screenshot where you're seeing "Queue plan"? (Make sure to leave out the top bar with any credentials, workspace/organization name, etc. that you don't want to share!) I'm seeing "Start new run" under the Actions menu, and I'm wondering if I'm not looking at the same thing you're looking at.

**Comment by sdehm** [comment]
Hello! Hope it helps. We are using Terraform Enterprise so maybe that is the discrepancy here? Here is a screenshot which also matches the screenshot gif on that same docs page at https://www.terraform.io/cloud-docs/run#retry-a-speculative-plan-in-the-ui but it also shows in the overall list of all "Runs".

image

**Comment by laurapacilio** [comment]
Thank you so much for sharing the screenshot! Because you're using Terraform Enterprise, I think is it likely that the version you're using has an older version of the UI than the most recent version of Terraform Cloud. We have been evolving the language we use to talk about operations in Terraform Cloud, and I know that there has been a shift from the "Queue plans" language to using "Run" to describe starting a process. 

The way our documentation is currently structured means that it will always show (or try to show!) the latest for Terraform Cloud. We are moving toward versioned documentation for Terraform Enterprise, which I'm hoping will help to prevent discrepancies for TFE users like the one you've highlighted here.

Because of that, I think we should actually change this to "Start new run". I know that's a bit odd, given that you don't see that in Terraform Enterprise, but I think it's the right thing (the current documentation does not reflect the most recent version of Terraform Cloud). Would you be willing to make that change to your PR? 

Let me know what you think/if you have questions!

File: content/cloud-docs/run/index.mdx [hunk 1]
```diff
@@ -63,7 +63,7 @@ Terraform Cloud has three main workflows for managing runs, and your chosen work
 - The [API-driven run workflow](/cloud-docs/run/api), which is more flexible but requires you to create some tooling.
 - The [CLI-driven run workflow](/cloud-docs/run/cli), which uses Terraform's standard CLI tools to execute runs in Terraform Cloud.
 
-You can initiate Terraform Cloud runs through the manual **Start new plan** action in the workspace actions menu, VCS webhooks, the standard `terraform apply` command (with the CLI integration configured), and [the Runs API](/cloud-docs/api-docs/run) (or any tool that uses that API).
+You can initiate Terraform Cloud runs through the manual **Queue plan** action in the workspace actions menu, VCS webhooks, the standard `terraform apply` command (with the CLI integration configured), and [the Runs API](/cloud-docs/api-docs/run) (or any tool that uses that API).
```
Comment (sdehm) [inline content/cloud-docs/run/index.mdx:66]:
```suggestion
You can initiate Terraform Cloud runs through the manual **Start new run** action in the workspace actions menu, VCS webhooks, the standard `terraform apply` command (with the CLI integration configured), and [the Runs API](/cloud-docs/api-docs/run) (or any tool that uses that API).
```

**Review by laurapacilio — APPROVED** [review]
Thank you!

**Pull request merged**